libraries/chain/steem_evaluator.cpp

```diff
@@ -2201,7 +2194,6 @@ void hf20_vote_evaluator( const vote_operation& o, database& _db )
       });
 
       /// if the current net_rshares is less than 0, the post is getting 0 rewards so it is not factored into total rshares^2
```

Comment:
This comment should follow the statement it was tied to or be removed if it no longer applies.